inc/log.class.php

```diff
@@ -228,15 +228,20 @@ static function history ($items_id, $itemtype, $changes, $itemtype_link = '', $l
          $new_value = Toolbox::substr($new_value, 0, 250);
       }
 
-      // Build query
-      $query = "INSERT INTO `glpi_logs`
-                       (`items_id`, `itemtype`, `itemtype_link`, `linked_action`, `user_name`,
-                        `date_mod`, `id_search_option`, `old_value`, `new_value`)
-                VALUES ('$items_id', '$itemtype', '$itemtype_link', '$linked_action',
-                        '".addslashes($username)."', '$date_mod', '$id_search_option',
-                        '$old_value', '$new_value')";
-
-      if ($DB->query($query)) {
+      $params = [
+         'items_id'          => $items_id,
+         'itemtype'          => $itemtype,
+         'itemtype_link'     => $itemtype_link,
+         'linked_action'     => $linked_action,
+         'user_name'         => $username,
```

Comment:
addslashes dropped ? you kept one here: https://github.com/glpi-project/glpi/pull/3250/files#diff-11a09748e657af84b23fd2599f1f3101R1248

Reply:
Indeed; this must stay